code/typescript/pipelines-workshop/test/hitcounter.test.ts

```diff
@@ -6,16 +6,13 @@ import { HitCounter } from '../lib/hitcounter';
 test('DynamoDB Table Created', () => {
     const stack = new cdk.Stack();
 
-    const tableCreateLambda = new lambda.Function(stack, 'TestFunction', {
-        runtime: lambda.Runtime.NODEJS_14_X,
-        handler: 'lambda.handler',
-        code: lambda.Code.inline('test')
-    });
```

Comment:
Moved the lambda initialization into the `HitCounter` initialization to match how the other tests are written.